APE26.rst

```diff
@@ -0,0 +1,331 @@
+Removing data storage (representations) from coordinate frames
+==============================================================
+
+Authors (alphabetical): Jeff Jennings, Adrian Price-Whelan, Nathaniel Starkman, Marten van Kerkwijk
+---------------------------------------------------------------------------------------------------
+
+:date-created: 2024 11 04
+:date-last-revised: 2025 04 28
+:date-accepted: 202x xx xx
+:type: Standard Track
+:status: Discussion
+
+Abstract
+--------
+Following the rationale presented in `APE 5 
+<https://github.com/astropy/astropy-APEs/blob/main/APE5.rst>`_, 
+coordinate frames in astropy.coordinates currently store metadata 
+used to construct the frame (i.e., for transforming between frames) 
+and may also store coordinate *data* itself. This duplicates 
+functionality with ``SkyCoord``, which acts as a container for both 
+coordinate data and reference frame information. We propose to 
+change the frame classes such that they only store metadata and 
+never coordinate data, superseding this aspect of the implementation 
+in `APE 5 <https://github.com/astropy/astropy-APEs/blob/main/APE5.rst>`_. 
+This would make the implementation more modular and performant, 
+remove ambiguity for users from having nearly duplicate functionality 
+with slightly different APIs, and better satisfy the principle of 
+`Separation of Concerns
+<https://en.wikipedia.org/wiki/Separation_of_concerns>`_.
+
+Detailed description
+--------------------
+The coordinate frame classes (subclasses of ``BaseCoordinateFrame``, 
+e.g., ``ICRS``, ``Galactic``, ``FK4``, etc.) are used to represent 
+astronomical reference frames. These may also contain position, 
+velocity, time, or other information about the reference frame 
+relative to another reference frame, which is used to transform 
+coordinates between the reference frames that exist in the 
+``astropy.coordinates`` ecosystem. For example, the ``AltAz`` frame 
+class, which is used when referencing sky coordinates in altitude 
+and azimuth, must contain a location on Earth and a time in order 
+to transform to and from an inertial reference frame like the ICRS. In 
+addition to reference frame information, these frame classes can 
+**also** contain coordinate data (positions, velocities, or other 
+differentials), which are stored internally using the 
+``Representation`` and ``Differential`` classes.
```

Comment:
There is nothing in `astropy.coordinates` called `Representation` or `Differential`.
```suggestion
``BaseRepresentation`` and ``BaseDifferential`` subclasses.
```